README.md

```diff
@@ -260,6 +260,7 @@ architecture model:
 | `generatr.style.colors.secondary`       | Secondary site color, used for font color in header bar and for active menu.                                                                                                                                                                                                         | `#cccccc` | `#ffffff`           |
 | `generatr.style.faviconPath`            | Site logo location relative to the configured `assets` folder. When configured, the logo image will be place on the left side in the header bar. This requires the `--assets-dir` switch when generating the site and the corresponding file to be available in the `assets` folder. |           | `site/favicon.ico`  |
 | `generatr.style.logoPath`               | Site favicon location relative to the configured `assets` folder. When configured, the favicon will be set for all generated pages. This requires the `--assets-dir` switch when generating the site and the corresponding file to be available in the `assets` folder.              |           | `site/logo.png`     |
+| `generatr.style.customCSSPath`          | Site custom css location relative to the configured `assets` folder. When configured this css file will be loaded for all pages. This requires the `--assets-dir` switch when generating the site and the corresponding file to be available in the `assets` folder.                 |           | `site/custom.css`   |
```

Comment:
What about having a configuration option that let's you set an url to a stylesheet, e.g. `generatr.style.includeUrl` or `generatr.style.iadditionalCssUrl`. That way the stylesheet can be hosted anywhere and we would just include that url in the page header.

Reply:
Let me see what I can do to allow either a url or a path to a local file. I think that would give the best of both worlds.

Reply:
Actually, thinking about how we handle favicon and logo, your proposal is fine. Reading a custom css from url would be very inconsistent with how we handle other resources. 
My only suggestion would be to rename the property to `generatr.style.customStylesheetPath` and rename function accordingly.

Reply:
Now supports both a local file or a remote resource. Also renamed function as suggested.